engine/src/main/java/org/terasology/rendering/assets/texture/AWTTextureFormat.java

```diff
@@ -120,4 +69,64 @@ public TextureData load(ResourceUrn urn, List<AssetDataFile> inputs) throws IOEx
     public PathMatcher getFileMatcher() {
         return path -> super.getFileMatcher().matches(path) && pathMatcher.matches(path);
     }
+
+    /**
+     * Converts a BufferedImage into a TextureData
```

Comment:
Trivial good practice: End the first sentence of javadoc descriptions with a `.`  :-)